### PR TITLE
feat(ui): add ActiveFiltersBar composable with tests

### LINKED ISSUE #117: Sub-Issue: Add UI element to display currently active filters.
### **Description**

Implement the UI for displaying displays currently active event filters (region, date range, availability) and allows users to clear them.

### **Acceptance Criteria**

* [ ] Active filters are displayed.
* [ ] A `Clear All` button is available to reset all filters.

### PR DESCRIPTION
**Pull request by walid-as**
# Description:
## Purpose of the change
This PR introduces the `ActiveFiltersBar` UI component to display currently active event filters.

## Related issues
Closes #117
Part of #90

## How to test
* Check out the branch.
 * Run unit tests in `ActiveFiltersBarTest`.
 * Verify all tests pass.
 
<p align="center">
  ActiveFiltersBar
</p>

### CODE REVIEW

File: app/src/androidTest/java/ch/onepass/onepass/ui/eventsFilters/ActiveFiltersBarTest.kt
Comment (M-e-d-A) [inline app/src/androidTest/java/ch/onepass/onepass/ui/eventsFilters/ActiveFiltersBarTest.kt]: **Nitpick:** Missing test: What happens when region is set to an empty string `""`? Should this be treated as no filter?

File: app/src/main/java/ch/onepass/onepass/ui/eventFilters/ActiveFiltersBar.kt [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import androidx.compose.ui.unit.dp
+import ch.onepass.onepass.model.eventFilters.EventFilters
+
+@Composable
+fun ActiveFiltersBar(
+    filters: EventFilters,
+    onClearFilters: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+  Surface(
```
Comment (M-e-d-A) [inline app/src/main/java/ch/onepass/onepass/ui/eventFilters/ActiveFiltersBar.kt]: **Nitpick:** The `modifier` parameter in `ActiveFiltersBar` is only used on the `Surface`. Consider if it should also propagate sizing constraints or if this is intentional.
